### ISSUE TITLE
[v3] Implement new Concept Exercise: properties

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/fsharp#843]
This issue describes how to implement the `properties` concept exercise for the F# track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student the basics of the Concept of Properties in F#.

## Learning objectives

- Know what properties are.
- Know how to define immutable properties.
- Know how to define mutable properties.
- Know how to define auto-implemented properties.
- Know how to get and set properties.
- Know how to use access control for properties.

## Out of scope

- Static properties.
- Indexed properties.
- Abstract properties.

## Concepts

The Concepts this exercise unlocks are:

- `properties`: know what properties are; know how to define immutable properties; know how to define mutable properties; know how to define auto-implemented properties; know how to get and set properties; know how to use access control for properties.

## Prerequisites

This exercise's prerequisites Concepts are:

- `classes`: know how to define classes.
- `access-control`: know what `public` and `private` are.

Any data types used in this exercise (e.g. `strings` or `integers`) should also be added as prerequisites.

### Hints

- [Properties][properties]: introduction to properties.
- [Properties syntax][properties-syntax]: the various ways in which properties can be defined.

### After

- [Properties][properties]: introduction to properties.
- [Properties syntax][properties-syntax]: the various ways in which properties can be defined.
- [Properties tutorial][properties-tutorial]: a properties tutorial.

## Representer

This exercise does not require any specific representation logic to be added to the [representer][representer].

## Analyzer

This exercise does not require any specific logic to be added to the [analyzer][analyzer].

## Implementing

To implement this exercise, please [follow these instructions](https://github.com/exercism/v3/blob/master/languages/fsharp/reference/implementing-a-concept-exercise.md).

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

[how-to-implement-a-concept-exercise]: https://github.com/exercism/v3/blob/master/docs/maintainers/generic-how-to-implement-a-concept-exercise.md
[implemented-exercises]: https://github.com/exercism/v3/tree/master/languages/fsharp/exercises/concept/README.md#implemented-exercises
[reference]: https://github.com/exercism/v3/blob/master/reference/concepts/classes.md
[analyzer]: https://github.com/exercism/fsharp-analyzer
[representer]: https://github.com/exercism/fsharp-representer
[meta-design]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/design.md
[meta-config.json]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/config.json
[docs-concept-exercises]: https://github.com/exercism/v3/blob/master/docs/concept-exercises.md
[properties]: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/members/properties
[properties-syntax]: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/members/properties#remarks
[properties-tutorial]: https://fsharpforfunandprofit.com/posts/classes/#properties
